pools-drained/src/agent.spec.ts

```diff
@@ -352,4 +352,33 @@ describe("Pool drained test suite", () => {
     const findings: Finding[] = await handler(tx);
     expect(findings).toStrictEqual([createFinding(v3.pair, v2.pair)]);
   });
+
+  it("should handle correctly tokens with fee on transfers", async () => {
+    const t0: string = createAddress("0x70");
+    const t1: string = createAddress("0x71");
+    const pair: string = utils.v2Create2(t0, t1);
+    const block: number = 15;
+
+    addV2Pair(pair, t0, t1, block);
```

Comment:
Not a big deal, but maybe change this to use `addV3Pair` as a `V3_IFACE` function is called?